javascript-source/games/adventureSystem.js

```diff
@@ -274,6 +276,7 @@
         }
 
         if (currentAdventure.gameState === 0) {
+            if (!$.checkUserPermission(username, undefined, startPermission)) return;
```

Comment:
New code must use curly braces regardless of previous code

Reply:
Added the curly braces.